app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Http\Requests\UpdateUserRequest;
+use App\Repositories\RepositoryInterface\UserRepositoryInterface;
+
+class UserController extends Controller
+{
+    protected $user;
+
+    public function __construct(UserRepositoryInterface $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $getUsers = $this->user->getAll();
```

Comment:
Đặt tên ngắn gọn dễ hiểu thôi e
```suggestion
        $users = $this->user->getAll();
```